src/data/array_group.js

```diff
@@ -44,6 +44,7 @@ class ArrayGroup {
         if (ElementArrayType2) this.elementArray2 = new ElementArrayType2();
 
         this.layerData = {};
+        this.paintPropertyStatistics = {};
```

Comment:
@jfirebaugh yeah, sure -- I actually started down that route, and then bailed because doing so would bifurcate the contents of `ArrayGroup#layerData` and the corresponding `BufferGroup#layerData`, and thought that might be (slightly) worse.   I don't have a strong opinion on this though, so deferring to you.